src/stylesheets/elements/typography/_paragraph.scss

```diff
@@ -0,0 +1,25 @@
+p {
```

Comment:
- Paragraph classes should use [body sizes](https://designsystem.digital.gov/utilities/font-size-and-family/) once `$theme-global-paragraph-styles` is set to true and [typography settings](https://designsystem.digital.gov/documentation/settings/#typography-settings) are set